### PR TITLE
Add pad lock icon

### PR DESCRIPTION
**Pull request by JoakimLofgren**
We need a locked pad lock icon for our "unauthorized page":

padlock

### CODE REVIEW

File: components/icons/PadLock.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+      <rect x='43.9358463' y='18.1132075' width='2.26415094' height='7.9245283'></rect>
+      <rect x='26.0377358' y='36.2264151' width='2.26415094' height='12.4528302'></rect>
+      <rect x='24.9056604' y='35.0943396' width='4.52830189' height='4.52830189' rx='2.26415094'></rect>
+    </g>
+  </Icon>
+)
+
+export default icon(PadLock)
+
```
Comment (joaomilho) [inline components/icons/PadLock.jsx]: we generally do don't keep a blank line in other files, do u have any reason to add one?

File: components/icons/PadLock.jsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+      <rect x='43.9358463' y='18.1132075' width='2.26415094' height='7.9245283'></rect>
+      <rect x='26.0377358' y='36.2264151' width='2.26415094' height='12.4528302'></rect>
+      <rect x='24.9056604' y='35.0943396' width='4.52830189' height='4.52830189' rx='2.26415094'></rect>
```
Comment (joaomilho) [inline components/icons/PadLock.jsx]: I suppose after the 1st or 2nd number after . there will be no diff to the human eye, even in a x8 display. We have been asking our designer to pass SVGs over https://jakearchibald.github.io/svgomg/, just to make sure.

File: components/icons/PadLock.jsx (hunk 1 above)
Comment (JoakimLofgren) [inline components/icons/PadLock.jsx]: Old habits die hard, want me to remove it?

**Comment by joaomilho** [comment]
So, for now you'll have to a) bump the verion b) push to master c) tag.

**Comment by JoakimLofgren** [comment]
@joaomilho According to the README I assume? 


> Send a PR to GitHub.
> Once approved:
>    Update the version using npm version (tag will have v prefix) & update CHANGELOG.md.
>    Merge to master and push (with the new tag as well).

File: components/icons/PadLock.jsx (hunk 2 above)
Comment (JoakimLofgren) [inline components/icons/PadLock.jsx]: Efficient tool :)

**Pull request merged**